core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -531,4 +557,103 @@ synchronized boolean isCompleted() {
          return stateStatus == StateStatus.COMPLETED;
       }
    }
+
+   @ManagedAttribute(description = "Number of keys that need to be sent to remote site(s)", displayName = "XsiteQueueSize", measurementType = MeasurementType.DYNAMIC)
+   public int getQueueSize() {
+      return updatedKeys.size();
+   }
+
+   @ManagedAttribute(description = "Number of Conflicts", displayName = "Number of Conflicts", measurementType = MeasurementType.TRENDSUP)
+   public long getNoOfConflicts() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return conflictsCounts.longValue();
```

Comment:
field `conflictsCounts` can be removed. Here, you can sum all the other 3 counters `numberOfConflics*`

Reply:
Push the changes